symfony/framework-bundle/3.3/public/index.php

```diff
@@ -21,7 +21,13 @@
     Debug::enable();
 }
 
-// Request::setTrustedProxies(['0.0.0.0/0'], Request::HEADER_FORWARDED);
+if ($trustedProxies = $_SERVER['APP_TRUSTED_PROXIES'] ?? false) {
+    Request::setTrustedProxies(json_decode($trustedProxies, true), Request::HEADER_X_FORWARDED_ALL);
```

Comment:
The `Request::HEADER_X_FORWARDED_ALL` would also need env-based configuration.
Can we autodetect AWS btw?

Reply:
Should we allow to set the list of headers to trust in a JSON or just let the user set the constant value directly?
How can we detect AWS securely (without letting an attacker usurp its identity in case the app run elsewhere)?

Reply:
let's use HEADER_X_FORWARDED_AWS_ELB by default! it's generally enough, and the most portable

Reply:
of be more explicit:
Request::HEADER_X_FORWARDED_ALL & ^Request::HEADER_X_FORWARDED_HOST